template/ios/Podfile

```diff
@@ -29,7 +33,9 @@ target 'HelloWorld' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable the next line.
-  use_flipper!()
+  if !IN_CI then
+    use_flipper!()
+  end
```

Comment:
Just a heads up that this would reduce our confidence on having Flipper working on the new app template. I'm not sure we want to do this.

On Android, we always include Flipper regardless of CI.

I can understand the win on build time for a RN user, but as we're testing the default template here, it probably makes sense to make sure Flipper is also included in the build